devtools/create_project/msvc.cpp

```diff
@@ -285,7 +285,7 @@ std::string MSVCProvider::getPostBuildEvent(MSVC_Architecture arch, const BuildS
 
 	cmdLine += (setup.useSDL2) ? "SDL2" : "SDL";
 
-	cmdLine += " &quot;%" LIBS_DEFINE "%/lib/";
+	cmdLine += " &quot;$(VcpkgCurrentInstalledDir)lib/";
```

Comment:
Looks like this isn't correct yet since DLLs are in the /bin folder, going to change it and test it out.